src/CosmosDB/CosmosDB/Models/PSVectorEmbedding.cs

```diff
@@ -0,0 +1,79 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.CosmosDB.Models;
+using Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.Models;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Commands.CosmosDB.Models
+{
+
+    public class PSVectorEmbedding
+    {
+        public PSVectorEmbedding()
+        {
+        }
+
+        public PSVectorEmbedding(VectorEmbedding vectorEmbedding )
+        {
+            if(vectorEmbedding == null)
+            {
+                return;
+            }
+
+            Path = vectorEmbedding.Path;
```

Comment:
i feel we should atleast do a basic null validation rather that go all the way to the BE and failing

Reply:
we had planed to add no validations